import_cli.adoc

```diff
@@ -0,0 +1,183 @@
+[#importing-a-managed-cluster-with-the-cli]
+= Importing a managed cluster with the CLI
+
+After you install multicluster engine for Kubernetes, you are ready to import a cluster to manage.
```

Comment:
Sug:  "...import a cluster to manage." -> "import a cluster so that you can manage it with the engine."

(I think its ok to refer to an installed instance of MCE simply as "the engine" vs. always repeating the MCE name.)

Reply:
I've discussed this with @swopebe-- my understanding is `the engine` or `the engine cluster` isn't currently acceptable (I had "the engine cluster" a bunch and started using the full name). 

For now let's use the full name. If @swopebe agrees there's a shorter name that we can agree to use after we say "multicluster engine for Kubernetes" the first time, then slimming it down should be easy.